src/array-repeat-strategy.js

```diff
@@ -109,11 +109,11 @@ export class ArrayRepeatStrategy {
         let {index, removed, addedCount} = splices[i];
         mergeSplice(repeat.__queuedSplices, index, removed, addedCount);
       }
-      repeat.__array = array.slice(0);
+      repeat.__array = array.slice(0, array.length);
```

Comment:
Well, in fact, according to this http://jsperf.com/new-array-vs-splice-vs-slice/113, `slice` is not the fastest, is at the medium of the table. Using a while loop seems to be faster, but is really worth?